modules/client-configuration/pages/clients-almalinux.adoc

```diff
@@ -3,12 +3,26 @@
 
 This section contains information about registering Salt clients running {almalinux} operating systems.
 
-[NOTE]
+ifeval::[{suma-content} == true]
+[IMPORTANT]
+====
+{amazon} repository URLs are available from {scc}, but the packages and metadata are provided by Amazon, not by {suse}.
+====
+endif::[]
+
+[IMPORTANT]
 ====
 Traditional clients are not available on {almalinux}{nbsp}8.
 {almalinux}{nbsp}8 clients are only supported as Salt clients.
 ====
 
+[WARNING]
+====
+{almalinux} instances always have the same `machine-id` id at `/etc/machine-id`.
```

Comment:
No, this is not true. This happens so far with the AlmaLinux created at AWS. Does not happen if you install (for example) on KVM using the ISO.